src/layouts/components/Footer.tsx

```diff
@@ -43,124 +43,146 @@ export const MenuItems = [
     ],
   },
   {
-    category: "ECOSYSTEM",
+    category: "RESOURCES",
     childLink: [
-      {
-        label: "DeFiChain Apps",
-        pathname: "/defichainapps",
-        testId: "defichainapps",
-        altLabel: "defichainapps",
-        isExternaLink: false,
-      },
-      {
-        label: "DeFi Meta Chain",
-        pathname: "/defimetachain",
-        testId: "defimetachain",
-        altLabel: "defimetachain",
-        isExternaLink: true,
-        url: "https://meta.defichain.com/",
-      },
       {
         label: "DeFiScan",
         pathname: "/defiscan",
         testId: "defiscan",
         altLabel: "defiscan",
         isExternaLink: true,
         url: "https://defiscan.live/",
+        hasUpArrowIcon: true,
       },
       {
-        label: "Governance",
-        pathname: "/governance",
-        testId: "governance",
-        altLabel: "governance",
-        isExternaLink: true,
-        url: "https://defiscan.live/governance",
-      },
-      {
-        label: "Bug Bounty",
-        pathname: "/bugbounty",
-        testId: "bugbounty",
-        altLabel: "bugbounty",
-        isExternaLink: false,
-      },
-      {
-        label: "$100M Accelerator Program",
-        pathname: "/100macceleratorprogram",
-        testId: "100macceleratorprogram",
-        altLabel: "100macceleratorprogram",
-        isExternaLink: false,
-      },
-    ],
-  },
-  {
-    category: "BUILD",
-    childLink: [
-      {
-        label: "Jellyfish SDK",
-        pathname: "/jellyfishsdk",
-        testId: "jellyfishsdk",
-        altLabel: "jellyfishsdk",
-        isExternaLink: true,
-        url: "https://jellyfishsdk.com/",
-      },
-      {
-        label: "DeFiChain White Paper",
-        pathname: "/defichainwhitepaper",
+        label: "White paper",
+        pathname: "/white-paper",
         testId: "defichainwhitepaper",
         altLabel: "defichainwhitepaper",
-        isExternaLink: false,
-      },
-      {
-        label: "GitHub",
-        pathname: "/github",
-        testId: "github",
-        altLabel: "github",
-        isExternaLink: true,
-        url: "https://github.com/DeFiCh/ain",
         hasUpArrowIcon: true,
       },
-      {
-        label: "Developer Resource",
-        pathname: "/developerResource",
-        testId: "developerResource",
-        altLabel: "developerResource",
-        isExternaLink: false,
-      },
-      {
-        label: "Media Assets",
-        pathname: "/mediaAssets",
-        testId: "mediaAssets",
-        altLabel: "mediaAssets",
-        isExternaLink: false,
-      },
-      {
-        label: "Security",
-        pathname: "/security",
-        testId: "security",
-        altLabel: "security",
-        isExternaLink: false,
-      },
     ],
   },
+  // {
```

Comment:
gonna remove? :O

Reply:
Nope!! At the moment, we are just hiding it because the sections are not ready. 
Will show again in the future so would not make sense to remove! < :